.cypress/integration/3_panels.spec.ts

```diff
@@ -10,578 +10,624 @@ import {
   delay,
   NEW_VISUALIZATION_NAME,
   PPL_FILTER,
+  PPL_VISUALIZATION_CONFIGS,
   PPL_VISUALIZATIONS,
   PPL_VISUALIZATIONS_NAMES,
   TEST_PANEL,
 } from '../utils/panel_constants';
 
-describe('Adding sample data and visualization', () => {
-  it('Adds sample flights data for visualization paragraph', () => {
+describe('Panels testing with Sample Data', () => {
+  suppressResizeObserverIssue();//needs to be in file once
+
+  before(() => {
     cy.visit(`${Cypress.env('opensearchDashboards')}/app/home#/tutorial_directory/sampleData`);
     cy.get('div[data-test-subj="sampleDataSetCardflights"]')
       .contains(/(Add|View) data/)
       .trigger('mouseover')
       .click();
-    cy.wait(delay * 3);
   });
-});
 
-describe('Creating visualizations', () => {
   beforeEach(() => {
```

Comment:
Each test will now delete all the saved dashboards and visuals to "cleanup" before executing its testable metric.